mobile/examples/Xamarin/readme.md

```diff
@@ -0,0 +1,59 @@
+# ONNX Runtime Xamarin Sample
+
+The [VisionSample](VisionSample/VisionSample.sln) demonstrates the use of several [vision-related models](https://github.com/onnx/models/tree/f064171f7dd8e962a8a5b34eac8e1bcf83cebbde#vision), from the [ONNX Model Zoo collection](https://github.com/onnx/models/tree/f064171f7dd8e962a8a5b34eac8e1bcf83cebbde#onnx-model-zoo), by a [Xamarin.Forms](https://dotnet.microsoft.com/apps/xamarin/xamarin-forms) app. 
+
+## Overview
+The sample enables you to take/pick a photo on the device or use a sample image, if one is provided, to explore the following models.
+
+### [ResNet](https://github.com/onnx/models/tree/f064171f7dd8e962a8a5b34eac8e1bcf83cebbde/vision/classification/resnet#resnet)
+
+Classifies the major object in the image into 1,000 pre-defined classes.
+
+### [Ultraface](https://github.com/onnx/models/tree/f064171f7dd8e962a8a5b34eac8e1bcf83cebbde/vision/body_analysis/ultraface#ultra-lightweight-face-detection-model)
+
+Lightweight face detection model designed for edge computing devices providing detection boxes and scores for a given image. 
+This model is included in the repository, but has been updated to remove unused initializers, and to make the initializers constant to enable more runtime optimizations to occur.
+
+The sample also demonstrates how to switch between the default **CPU EP ([Execution Provider](https://onnxruntime.ai/docs/execution-providers))** and platform-specific options. In this case, [NNAPI](https://onnxruntime.ai/docs/execution-providers/NNAPI-ExecutionProvider.html) for Android and [Core ML](https://onnxruntime.ai/docs/execution-providers/CoreML-ExecutionProvider.html) for iOS.
+
+## Getting Started
+
+> [!IMPORTANT] 
+> The nuget.config has package sources for the official and integration NuGet repositories. 
+> Xamarin support is currently only available in the ONNX Runtime nuget packages in the integration repository. It will be added to the official package in the v1.10 release.
+>
+> There are some [known issues](#known-issues) that could impact aspects of the sample on specific devices or environments. See [known issues section](#known-issues) for workarounds.
+
+The [VisionSample](VisionSample/VisionSample.sln) should build and run as-is, and looks for model files in a folder in this directory called **Models**. 
+The Ultraface model should already exist in that directory and to able to be used.
```

Comment:
Not sure why include the Ultraface model in the directory but the ResNet need to be downloaded? Thought might be nice to do it in a consistent way.

Reply:
Ultra face is small. ResNet is nearly 100MB